reporters/SponsorReporter.js

```diff
@@ -79,15 +80,20 @@ class SponsorReporter {
     const allPositions = this.empClient.getAllPositions();
     const currentPrice = this.priceFeed.getCurrentPrice();
 
+    // Define the table column headings before hand to re-use the variables.
+    const col1Heading = `Synthetic debt(${this.empProps.syntheticCurrencySymbol})`;
+    const col2Heading = `Backing collateral(${this.empProps.collateralCurrencySymbol})`;
+    const col3Heading = "Position CR %";
+
+    // Place holder object to store all table information.
     let allSponsorTable = {};
     for (let position of allPositions) {
-      allSponsorTable[position.sponsor] = {
-        "Collateral(DAI)": this.formatDecimalString(position.amountCollateral),
-        "Tokens borrowed(ETHBTC)": this.formatDecimalString(position.numTokens),
-        "Position CR %": this.formatDecimalString(
-          this._calculatePositionCRPercent(position.amountCollateral, position.numTokens, currentPrice)
-        )
-      };
+      allSponsorTable[position.sponsor] = {};
+      allSponsorTable[position.sponsor][col1Heading] = this.formatDecimalString(position.numTokens);
+      allSponsorTable[position.sponsor][col2Heading] = this.formatDecimalString(position.amountCollateral);
+      allSponsorTable[position.sponsor][col3Heading] = this.formatDecimalString(
+        this._calculatePositionCRPercent(position.amountCollateral, position.numTokens, currentPrice)
+      );
```

Comment:
nit: I think this could be done very similarly to how it was done previously:
```
allSponsorTable[position.sponsor] = {
   [col1Heading]: this.formatDecimalString(position.numTokens);
...
}
```

Right?

Reply:
Ye, that's nice and clean. I've implemented it as such :)